webapp/pages/post/_id/_slug/more-info.vue

```diff
@@ -62,6 +63,9 @@ export default {
     post() {
       return this.Post ? this.Post[0] || {} : {}
     },
+    sortedTags() {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-03T14:52:39Z" title="Wednesday, June 3rd 2020, 4:52:39 pm +02:00">Jun 3, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -62,6 +62,20 @@ export default {
     post() {
       return this.Post ? this.Post[0] || {} : {}
     },
+    sortedTags() {
```


Would be nice to have a shared function … see above ☝🏼